### PR TITLE
Add a(n E2E) test for basic (left) arrow key usage

### LINKED ISSUE #8483: (Multiple) webElement.sendKeys(Keys.LEFT) do(es)n't work on a Quill(JS.com) editor
## 🐛 Bug Report

The selection (position) doesn't change.

## To Reproduce

Detailed steps to reproduce the behavior:

```
editor.sendKeys("bfmtv.frest");
for (byte j = 0; j < 3; j++) 
	editor.sendKeys(Keys.LEFT);
editor.sendKeys(" ");
```

## Expected behavior

The cursor should move left by 2, but doesn't. The next character typed is positioned as if these left keys hadn't been pressed at all. Result: 
"bfmtv.frest "
instead of 
"bfmtv.fr est"

## Environment

OS: Linux
Browser: Chrome
Browser version: 83
Browser Driver version: 83.0.4103.39 (ccbf011cb2d2b19b506d844400483861342c20cd-refs/branch-heads/4103)

Language Bindings version: Java 3.13.0 

## Work-around
I used Quill's .updateContents(...) instead to send an event simulating what's happening in real user session.

### PR DESCRIPTION
**Pull request by haqer1**
Fixes: #8483 

### Description
Helps ensure that Selenium supports basic (left) arrow key usage

### Motivation and Context
I used this to determine that the issue i've worked around stems from Quill(JS.com), rather than Selenium

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: java/client/test/org/openqa/selenium/interactions/BasicKeyboardInterfaceTest.java [hunk 1]
```diff
@@ -297,6 +300,22 @@ public void testSelectionSelectAll() {
     assertThat(input.getAttribute("value")).isEqualTo("");
   }
 
+  @Test
+  public void testLeftArrowEntry() {
+    driver.get("https://google.com/ncr");
```
Comment (AutomatedTester) [inline java/client/test/org/openqa/selenium/interactions/BasicKeyboardInterfaceTest.java]: Can you add a new html page to `common/src/web` and use that instead

File: java/client/test/org/openqa/selenium/interactions/BasicKeyboardInterfaceTest.java [hunk 2]
```diff
@@ -45,6 +45,9 @@
  * Tests interaction through the advanced gestures API of keyboard handling.
  */
 public class BasicKeyboardInterfaceTest extends JUnit4TestBase {
+  static final String LEFT_ARROW_SPACE_TEST_STRING_CORE = "bfmtv";
```
Comment (AutomatedTester) [inline java/client/test/org/openqa/selenium/interactions/BasicKeyboardInterfaceTest.java]: Add these to the test method so it's all together

**Review by AutomatedTester — CHANGES_REQUESTED** [review]

**Comment by AutomatedTester** [comment]
just going to wait for travis before merging

**Comment by haqer1** [comment]
> just going to wait for travis before merging

Cool.

P.S. FYI, i re-used _single_text_input.html_ (re-use of code that already exists is frequently the best thing to do, & seems to be the case for the page aspect here).

**Review by AutomatedTester — APPROVED** [review]

**Review by diemol — APPROVED** [review]
Thank you for your contribution @haqer1!

**Pull request merged**